docs/reference/release-notes/highlights.asciidoc

```diff
@@ -17,39 +17,16 @@ Other versions:
 
 // The notable-highlights tag marks entries that
 // should be featured in the Stack Installation and Upgrade Guide:
-
 // tag::notable-highlights[]
-[discrete]
-[[integrate_filtering_support_for_approximate_nearest_neighbor_search]]
-=== Integrate filtering support for approximate nearest neighbor search
+// [discrete]
```

Comment:
I think we generally only update the "What's New" page for major releases, so the changes to this file seem incorrect.

Reply:
Yes, I agree.  Highlights for 8.2.0 should not have been deleted as part of the process of generating the 8.2.2 release notes.

Reply:
I'll create a PR to re-add this info